src/MAKE/OPTIONS/Makefile.intel_cpu_openmpi

```diff
@@ -7,13 +7,14 @@ SHELL = /bin/sh
 # specify flags and libraries needed for your compiler
 
 export OMPI_CXX = icc
-CC =		mpicxx -std=c++11 -diag-disable=10441 -diag-disable=2196
+CC =		mpicxx -std=c++11 -diag-disable=10441 -diag-disable=2196 -diag-disable=11074 -diag-disable=11076
 OPTFLAGS =      -xHost -O2 -fp-model fast=2 -no-prec-div -qoverride-limits \
                 -qopt-zmm-usage=high
 CCFLAGS =	-qopenmp -qno-offload -ansi-alias -restrict \
                 -DLMP_INTEL_USELRT -DLMP_USE_MKL_RNG $(OPTFLAGS) \
                 -I$(MKLROOT)/include
 SHFLAGS =	-fPIC
+FMTFLAGS = 	-std=c++11
```

Comment:
Why is it c++14 in CMake and C++11 in the Makefiles?

Reply:
Because in CMake we can add this hack selectively to affected compiler versions and when the required standard has been raised to C++17 by inclusion of the KOKKOS package. Lowering to C++14 is sufficient.
In the make based build this flag is *always* applied when using the modified makefiles, even when KOKKOS is not installed, So it has to be compatible with the C++11 requirement for core LAMMPS. Installing the KOKKOS package raises the requirement to C++17 through modifying `Makefile.package.settings` and that needs to be cancelled.